src/main.rs

```diff
@@ -1,3 +1,7 @@
+#![cfg_attr(not(debug_assertions), windows_subsystem = "windows")] // hide console window on Windows in release
+
+use dials_research::run;
```

Comment:
would prefer just `dials_research::run()`, the import seems a little wierd

Reply:
Yes